server/nri-api.go

```diff
@@ -69,6 +69,16 @@ func (a *nriAPI) runPodSandbox(ctx context.Context, criPod *sandbox.Sandbox) err
 	return err
 }
 
+func (a *nriAPI) updatePodSandbox(ctx context.Context, criPod *sandbox.Sandbox) error {
+	if !a.isEnabled() {
+		return nil
+	}
+
+	pod := nriPodSandbox(ctx, criPod)
+
+	return a.nri.UpdatePodSandbox(ctx, pod)
+}
+
```

Comment:
I think we need to update this to get extra arguments for the updated pod `overhead` and `resources` and then pass them on to NRI. Something like this:

```
func (a *nriAPI) updatePodSandbox(ctx context.Context, criPod *sandbox.Sandbox, overhead, resources *cri.LinuxContainerResources) error {
	if !a.isEnabled() {
		return nil
	}

	pod := nriPodSandbox(ctx, criPod)

	return a.nri.UpdatePodSandbox(ctx, pod, api.FromCRILinuxResources(overhead), api.FromCRILinuxResources(resources))
}
```